QgisModelBaker/libqgsprojectgen/db_factory/db_factory.py

```diff
@@ -123,14 +123,15 @@ def get_specific_messages(self):
 
         return messages
 
-    def customize_widget_editor(self, field: Field, data_type: str):
+    def customize_widget_editor(self, field: Field, data_type: str, fielddef: dict):
         """Allows customizing the way a field is shown in the widget editor.
 
         For instance, a boolean field can be shown as a checkbox.
 
         :param field: The field that will be customized
         :type field: :class:`Field`
         :param data_type: The type of field
+        :param fielddef: The field info from database
```

Comment:
Can you add more information about the expected/supported content of the dict?
Or possibly add a `class FieldDef` instead of the dict which defines the supported properties?

Reply:
Done!